main.js

```diff
@@ -1,4 +1,5 @@
 const bookForm = document.querySelector('#form');
+const collection = document.querySelector('.table');
 
 function Book() {
   this.title = document.querySelector('#title').value;
```

Comment:
- [ ] Following the project requirements, you use classes instead of arrays and objects.
```javascript
class Books {
  constructor() {
  }
}
```
- [ ] You should also create Create class methods to add and remove books. Check out [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) to learn more about **classes**.